Configuration/PyReleaseValidation/README.md

```diff
@@ -61,6 +61,8 @@ The offsets currently in use are:
 * 0.7: trackingMkFit modifier
 * 0.701: DisplacedRegionalStep tracking iteration for Run-3
 * 0.702: trackingMkFit modifier for Phase-2 (initialStep only)
+* 0.703: LST tracking, initialStep+HighPtTripletStep only, on CPU
+* 0.704: LST tracking, initialStep+HighPtTripletStep only, on GPU
```

Comment:
Another thing: would it make sense to implement a "GPU vs CPU" workflow, similar to the `xxx.403` ones for Run 3 ?

Reply:
good idea for a todo in some followup PR. Or did you mean it has to be now?

Reply:
It's fine for me if this is added in a later PR.